qutip/core/qobj.py

```diff
@@ -162,8 +160,12 @@ class Qobj:
 
     Attributes
     ----------
-    data : array_like
-        Sparse matrix characterizing the quantum object.
+    data : object
```

Comment:
`data` should be an instance of `qutip.core.data.Data` and not an arbitrary object (minor nitpick, but might help users understand what this is).